x-pack/plugin/src/test/resources/rest-api-spec/api/security.authenticate.json

```diff
@@ -1,10 +1,12 @@
 {
```

Comment:
I've seen the rest of the PRs are not touching the file names for these, but it made sense to me. No problem to revert if there are objections though.